divi/src/wallet.h

```diff
@@ -34,6 +31,7 @@
 #include <utility>
 #include <vector>
 #include <boost/foreach.hpp>
+#include <main.h>
```

Comment:
You reverted the changes to move the methods and reintroduced main.h even though it was preferable not to include it in the header.

Reply:
Suggest removing main and keeping the two functions that the wallet file needs locally

Reply:
done , another #include <main.h> removed